.github/ISSUE_TEMPLATE.md

```diff
@@ -1,34 +1,47 @@
-<!--- Provide a general summary of the issue in the Title above -->
-
-Your issue may already be reported!
-Please search on the [issue track](../) before creating one.
-
-## Expected Behavior
 <!--- If you're describing a bug, tell us what should happen -->
 <!--- If you're suggesting a change/improvement, tell us how it should work -->
+## Expected Behavior
+
+
+
```

Comment:
I'm a little concerned with leaving so much whitespace between each section.  Users might think they're done filling out the form if they don't see text that falls below the fold of the textarea.  Can we consider leaving fewer spaces between each section?

Reply:
What's your suggestoin? 2-3 lines? 
If they're too close up on eachother it's quite overwhelming/chaos

Reply:
![](https://i.imgur.com/35GNjx0.png)

This is the current overview - within github the spaces look really wide due to the changes in between. I'm cool with moving it to 3 spaces instead of 4

Reply:
![](
![image](https://user-images.githubusercontent.com/10922045/41834786-bbd73aa2-7855-11e8-9bc8-7100e4619ade.png)
)